app/models/manageiq/providers/ansible_tower/provider.rb

```diff
@@ -5,6 +5,14 @@ class ManageIQ::Providers::AnsibleTower::Provider < ::Provider
           :dependent   => :destroy,
           :autosave    => true
 
+  has_many :endpoints, :as => :resource, :dependent => :destroy, :autosave => true
+
+  delegate :url,
+           :url=,
+           :to => :default_endpoint
+
+  virtual_column :url, :type => :string, :uses => :endpoints
```

Comment:
That's so weird...I wonder why we don't have that issue with ExtManagementSystem.  Maybe, it's this line

> class 'Endpoint' is associated to 'ExtManagementSystem' via :has_many.

That implies to me that we are not doing the reverse polymorphic properly, but I'm not sure.